sdk/inc/azure/iot/az_iot_common.h

```diff
@@ -61,6 +61,18 @@ typedef enum
   AZ_IOT_STATUS_TIMEOUT = 504,
 } az_iot_status;
 
+/**
+ * @brief Supported IoT Hub message properties
+ */
```

Comment:
Public header file change/new feature. Add to changelog?

Reply:
added. plz take a look